docs/xls-70d-on-chain-credentials/index.md

```diff
@@ -0,0 +1,47 @@
+
+The design of On-Chain Credentials draws from the [W3C Verifiable Credentials standard](https://www.w3.org/TR/vc-data-model-2.0/) and is intended to be compatible to an extent that makes sense in the context of the XRP Ledger. There are some differences in data structures and formatting: for example, the subject of a credential is identified by an XRP Ledger address rather than a URL.
+
+## Overview
+
+_Credentials_ are signed statements that can be stored in the ledger (in other words, on-chain) and can attest to a user's identity, legal status, or other status. This feature includes issuance, storage, and verification of credentials directly on the XRP Ledger, while still supporting the privacy needs of users.
+
+This feature set is designed for an ecosystem of parties with the following roles:
+
+- _Authorizers_ who want to limit specific actions or interactions to users who possess the right credentials.
+- _Issuers_ who give out credentials to users who meet their criteria.
+- _Users_ who obtain credentials from issuers and use those credentials to interact with authorizers.
+
+Each credential applies to a specific user's XRP Ledger account, and is attested by a specific credential issuer. The exact qualifications that a credential entails are open-ended, and defined by their issuers. They can range from trivial to serious, or broad to narrow. Some examples of things a credential could state:
+
+- That a user is not a subject of sanctions by a particular nation or set of nations.
+- That a user has accredited investor status in the US.
+- That a user has reached a specific level in a particular video game.
+
+### Uses
+
+Within the XRP Ledger, you can use Deposit Authorization to automatically allowlist any senders with the correct set of credentials, instead of approving sending accounts individually. Future amendments could further extend credentials to gate access to other features such as holding tokens, participating in lending pools, or contributing to an AMM.
+
+Credentials stored on-chain can also be used to authorize off-chain activities, especially in association with a decentralized identifier.
+
+## Usage Flow
+
+The typical usage for Credentials features three parties with different roles. Consider the following example:
+
+* Verity is a regulated business that wants to interact only with properly KYC'd accounts, to ensure legal compliance. This makes Verity an _authorizer_ because they configure which accounts are allowed (authorized) to interact with them.
```

Comment:
Do we want to use a fake biz name here instead?

Reply:
I just reused what was in the spec, but is Verity not a fake biz name? Sounds like one to me.

Reply:
It was originally a fake person name, actually. Verity the Verifier.